src/lib.rs

```diff
@@ -87,7 +87,7 @@ mod vs_instances;
 #[cfg(windows)]
 mod windows_sys;
 
-pub mod windows_registry;
+pub mod windows_helper;
```

Comment:
This is a breaking change, I suggest you to rename it baxk to `windows_registry`

Reply:
**resolved**